api/service/explorer/structs.go

```diff
@@ -28,6 +28,17 @@ type Address struct {
 	TXs     []*Transaction `json:"txs"`
 }
 
+// Committee ...
```

Comment:
Can you further document both this and `Validator` below?  This is an outward-facing data structure to be JSON-serialized and served to the client, so they better be fully documented for the API users.